sql/create.sql

```diff
@@ -59,3 +59,10 @@ create table shopidentifyer.stripe_cache (
   subscription varchar(255),
   stripe_status varchar(255)
 );
+
+create table shopidentifyer.admin_users (
+    stripe_id PRIMARY KEY,
+    password varchar(2048)
```

Comment:
bcrypt's hashes will always be 60 chars long. Safe to make this much smaller than 2048